packages/react-form-renderer/src/components/form-renderer.js

```diff
@@ -10,9 +10,35 @@ import defaultSchemaValidator from '../parsers/default-schema-validator';
 import SchemaErrorComponent from '../form-renderer/schema-error-component';
 import defaultValidatorMapper from '../validators/validator-mapper';
 
+const useComponentSpy = (onStateUpdate) => {
```

Comment:
What about to move this to hooks folder?

Reply:
I could. If you reaaaaaally want me to

Reply:
Yes, please :D